### PR TITLE
Add simple test as POC

### PR DESCRIPTION
**Pull request by rgoliveira**
I added a simple test case to serve as a proof-of-concept. It's a bit hack-ish since I just wanted to make sure this was even doable. And turns out it is! :tada:

I had to put a `setInterval` in there to wait for the bot to be ready. I'm accepting suggestions on how to make this more clear (maybe with promises?).

For the tests, I'm using:
- [`tape`][1] as the test harness
- [`tap-summary`][2] to pretty-print the test results

[1]: https://github.com/substack/tape
[2]: https://github.com/zoubin/tap-summary

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -3,4 +3,4 @@ sudo: false
 dist: trusty
 
 script:
-  - npm run lint .
+  - npm run lint . && npm test
```
Comment (raylu) [inline .travis.yml]: you can change this to just be
```yaml
script:
  - npm test
  - npm run lint
```

(we probably want to run tests first)

Comment (rgoliveira) [inline .travis.yml]: Thanks for the feedback.

That was intended. My rationale was that eslint tend to catch simple mistakes we sometimes overlook, and also forces the code format before anyone can run any tests.

In your local environment you can still do linting and testing as you wish, of course.

Was it a bad call?

**Comment by rgoliveira** [comment]
Related issue: #40

File: .travis.yml (hunk 1 above)
Comment (raylu) [inline .travis.yml]: Why would you not want to see the output of both anyway? You will still see ESLint errors if there are any, but this way you will see test failures too. (Both sections are collapsible in the webface.) The code still has to pass ESLint to pass the build.

Comment (rgoliveira) [inline .travis.yml]: Ah, I thought it would stop in the first non zero return. I'll change it later today. Thanks

Comment (raylu) [inline .travis.yml]: Er, I thought that's what you wanted. `&&` doesn't execute the right side if the left side exits non-zero.

**Pull request merged**